docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/heroku.md

```diff
@@ -230,7 +230,7 @@ heroku config:set NODE_ENV=production
 
 ### 5. Create your Strapi server config for production
 
-Create a new `server.js` in a new [env](/developer-docs/latest/setup-deployment-guides/configurations/optional/environment.md) folder. In this file you only need one key, the `url`, to notify Strapi what our public Heroku domain is. All other settings will automatically be pulled from the default `./config/server.js`.
+Create a new `server.js` in a new [env](/developer-docs/latest/setup-deployment-guides/configurations/optional/environment.md) folder. In this file you only need one key, the `url`, to notify Strapi what our public Heroku domain is. All other settings will automatically be pulled from the default `./config/server.js`. 
```

Comment:
Sentence after Heroku that mentions the session middleware "APP_KEYS", please kindly elaborate.

Reply:
Sorry, I should have written `app.keys`. I was thinking something like: " Additionally, the session middleware `keys` property must be set."  We want to make sure that the changes in the code block are accurately summarized in this introductory paragraph. There might be better phrasing but this will work I think.